packages/proxy/lib/http/util/cookies.ts

```diff
@@ -19,15 +60,15 @@ interface RequestDetails {
 // see https://github.com/salesforce/tough-cookie#samesite-cookies
 export const getSameSiteContext = (autUrl: string | undefined, requestUrl: string, isAUTFrameRequest: boolean) => {
```

Comment:
```suggestion
export const getSameSiteContext = (autUrl: string | undefined, requestUrl: string, isAUTFrameRequest: boolean): Protocol.Network.CookieSameSite => {
```

Reply:
I'm not sure we want to pull the CDP types into the proxy, but we could also make a reserved type for this even though it is currently implied